src/main/app/claims/models/claim.ts

```diff
@@ -198,4 +203,20 @@ export class Claim {
   private isSettlementReached (): boolean {
     return this.settlement && !!this.settlementReachedAt
   }
+
+  private isCCJPaidWithinMonth (): boolean {
+    let futureMonth
+
+    if (this.countyCourtJudgmentIssuedAt) {
+      futureMonth = calculateMonthIncrement(this.countyCourtJudgmentIssuedAt)
+    } else if (this.countyCourtJudgmentRequestedAt) {
+      futureMonth = calculateMonthIncrement(this.countyCourtJudgmentRequestedAt)
+    }
+
+    if (this.moneyReceivedOn.isSameOrBefore(futureMonth)) {
```

Comment:
`futureMonth` could still be null if neither `countyCourtJudgmentIssuedAt` nor `countyCourtJudgmentRequestedAt` are defined

Reply:
Could also be simplified to `return this.moneyReceivedOn.isSameOrBefore(futureMonth)`

Reply:
countyCourtJudgmentIssuedAt and countyCourtJudgmentRequestedAt are being checked to see if they are null or not (if a CCJ exists). If they are null isCCJPaidWithinMonth() should return false. 

Yes that return statement is a good way to simplify it cheers!

Reply:
If we are confident that at least one of the two will be present, I suggest using `if (this.countyCourtJudgmentIssuedAt) { ... } else { ... }`, so that if there is ever a bad refactoring causing that to change, the problem will be thrown from a more representative stack of code.